src/main/java/uk/gov/justice/digital/hmpps/cmd/api/repository/ShiftNotificationRepository.kt

```diff
@@ -13,6 +13,6 @@ interface ShiftNotificationRepository : CrudRepository<ShiftNotification, Long>
 
     fun findAllByProcessedIsFalse(): Collection<ShiftNotification>
 
-    fun countAllByQuantumIdAndShiftDateAndShiftType(quantumId: String, shiftDateTime: LocalDateTime, shiftNotificationType: String): Int
+    fun countAllByQuantumIdAndShiftDateAndShiftTypeAndShiftModified(quantumId: String, shiftDateTime: LocalDateTime, shiftNotificationType: String, shiftModified: LocalDateTime): Int
```

Comment:
We have to add shift modified into this check, otherwise we deduplicate multiple edits to the same shift, negating the work done in sending where filter out all but the latest.
show all on the page, only send the latest.